utils/tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-{flake8,unit}
+    py{27,35}-{flake8,unit,pylint}
```

Comment:
Main reason was to allow for calling out the tests separately like we currently allow in the Makefile.

That said, I think it would be valuable to have many of the tests re-use the same virtualenv vs a virutalenv for each and every environment target.